internal/core/entities/base.go

```diff
@@ -8,10 +8,10 @@ import (
 )
 
 type DBModel struct {
-	ID         uuid.UUID `gorm:"primaryKey;type:uuid" json:"id"`
+	ID         uuid.UUID `gorm:"primaryKey;type:uuid;Column:id" json:"id"`
```

Comment:
Column is used when the name defined in the struct is different than the name of the column. in this case there is no need for this.

I suggest undoing all unnecessary changes to the files that you are not the author of.